devito/ir/stree/algorithms.py

```diff
@@ -35,9 +36,13 @@ def stree_schedule(clusters):
     """
     stree = ScheduleTree()
 
-    prev = None
+    prev = Cluster(None)
     mapper = DefaultOrderedDict(lambda: Bunch(top=None, bottom=None))
 
+    def reuse_metadata(c0, c1, d):
+        return (c.guards.get(d) == prev.guards.get(d) and
```

Comment:
what I am i missing?
should not c/prev be c0/c1 ?

Reply:
indeed, CI failing miserably ! copy-paste mistake in my last commit. Fixing now...